### DISCUSSION TITLE
SqlSync with Dioxus?

### DISCUSSION THREAD

**Discussion by frederikhors** [orbitinghail/sqlsync#56] [Q&A]
I was wondering, dear guys, if any of you have tried sqlsync with dioxus?

**Comment by carlsverre** [comment]
I haven't yet, but it's such a great idea! I'll have cycles to work on SQLSync in the upcoming months at which point I'd love to expand it beyond just the browser. That said, if anyone has cycles to experiment I think Dioxus would be a natural fit for SQLSync!
